### ISSUE TITLE
SEC-004 : Generate release keystore (RSA 4096, 100y) with encrypted backup runbook

### ISSUE THREAD

**Issue by RajuSudhar** [RajuSudhar/nr-latch#19]
## Issue ID
SEC-004

## Summary
Generate keystore: `keytool -genkey -keyalg RSA -keysize 4096 -validity 36500 -alias nrlatch-release`. Document encrypted backup to ≥2 locations + keystore-loss runbook (loss = WRITE_SECURE_SETTINGS revocation for all installs). Wire `signingConfigs` reading from `~/.gradle/gradle.properties` (NOT committed).

## Reference Documents
- **Primary doc**: [`docs/architecture/04-security-hardening.md`](../blob/endgame/docs/architecture/04-security-hardening.md)
- **Section**: `## Build config hardening`
- **ADRs**: None
- **Cross-cuts**: None

## Acceptance Criteria
- [ ] Keystore generated
- [ ] Backup verified
- [ ] Signing wired
- [ ] Release APK signed

## Dependencies
- **Blocks**: #25, #26
- **Blocked by**: #5

## Verification
\`\`\`bash
apksigner verify app-release.apk
\`\`\`

## Implementation Notes
RSA 4096, 100-year validity. Encrypted backup runbook. gradle.properties (local, NOT committed).

## Priority / Size
P0 / S
